packages/curve-ui-kit/src/themes/components.ts

```diff
@@ -79,6 +79,11 @@ export const createComponents = (design: DesignSystem): ThemeOptions['components
       },
     },
   },
+  MuiSnackbar: {
+    defaultProps: {
+      autoHideDuration: 6000,
```

Comment:
I'd recommend against this because in the future when we have more important transaction related snackbars I'd like to not accidentally forget to reconfigure it. If a tx fails I'd be highly annoyed if the important error message disappears.